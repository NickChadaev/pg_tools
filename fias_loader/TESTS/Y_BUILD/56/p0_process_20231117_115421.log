2023-11-17 11:54:21 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_56 postgres /home/n.chadaev@abrr.local/Y_BUILD/56/stage_gar_c_56.csv 2023-11-13
--------------------------------------------------------
[11:54:21] ... Оренбургская обл., id_region: 56, Execute: /home/n.chadaev@abrr.local/Y_BUILD/56/stage_gar_c_56.csv, process_id:25345
[11:54:21] ... Start process
[11:54:21] ...  -- Version
[11:54:21] ...  -- Очистка от данных
[11:54:22] ...  -- Убираем индексы
[11:54:22] ...  -- Set UNLOGGED
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[11:54:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_REESTR_OBJECTS_20231113_2736dcf7-b639-414e-a5db-c4d5414b7e7f.XML
[12:09:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_ADDR_OBJ_20231113_0de4c16b-2b50-4f16-a807-db6fc261e5e2.XML
[12:09:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_HOUSES_20231113_ccdbaf93-09e7-493a-b9d1-51bba15f89b4.XML
[12:23:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_STEADS_20231113_dd33e2de-0f4a-405c-8fff-50ea72101203.XML
[12:24:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_APARTMENTS_20231113_a1f0998b-048a-4282-aba4-e4e5ae28abf6.XML
[12:31:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_ROOMS_20231113_3241774f-d3ea-42df-b23f-8273ac201866.XML
[12:31:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_CARPLACES_20231113_5f9cd966-7edf-46c0-9987-39fc99c84927.XML
[12:31:59] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_ADM_HIERARCHY_20231113_a4f66255-3e88-477e-8fc1-f42b864d4ca9.XML
[12:49:31] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_MUN_HIERARCHY_20231113_103d63ef-84b1-4065-a446-e5484fbfc168.XML
[13:06:07] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_CHANGE_HISTORY_20231113_173fcd29-6b79-4fd2-8eaa-100086fa2bee.XML
[13:50:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_ADDR_OBJ_DIVISION_20231113_3be4ed39-779f-4b19-8ea5-2e0dcf358267.XML
[13:50:52] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_ADDR_OBJ_PARAMS_20231113_93c3f4bc-d078-4fe2-bf94-213d4c9e2e2c.XML
[13:54:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_STEADS_PARAMS_20231113_5aea4abe-22b5-4ea3-9cb4-660764870b8a.XML
[14:04:00] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_HOUSES_PARAMS_20231113_45acd324-0dc2-4a35-b0b4-accde9179691.XML
[15:32:34] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_APARTMENTS_PARAMS_20231113_9b1c5291-12ea-47ec-b756-b14a56cc2893.XML
[15:44:37] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_ROOMS_PARAMS_20231113_209f7e33-6962-490f-b1b2-e74f4a60792c.XML
[15:44:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_CARPLACES_PARAMS_20231113_e6090582-0bcd-43fe-97c6-0e9a3f35a523.XML
[15:44:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/56/AS_NORMATIVE_DOCS_20231113_7cc982a5-85df-435f-b4c9-719f7535e9b1.XML
[15:47:21] ...  -- Data finish
[15:47:21] ...  -- Set LOGGED
[15:47:53] ... Stop process
--------------------------------------------------------
