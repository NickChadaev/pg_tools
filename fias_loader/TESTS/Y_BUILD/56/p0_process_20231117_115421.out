 save_gar_version 
------------------
               81
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                81
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1189732
gar_fias.as_addr_obj: 27775
gar_fias.as_houses: 964723
gar_fias.as_steads: 68787
gar_fias.as_apartments: 544492
gar_fias.as_rooms: 8551
gar_fias.as_carplaces: 108
gar_fias.as_adm_hierarchy: 1283194
gar_fias.as_mun_hierarchy: 1220494
gar_fias.as_change_history: 3606291
gar_fias.as_addr_obj_division: 100
gar_fias.as_addr_obj_params: 328300
gar_fias.as_steads_params: 726142
gar_fias.as_houses_params: 7099793
gar_fias.as_apartments_params: 960109
gar_fias.as_rooms_params: 9537
gar_fias.as_carplaces_params: 108
gar_fias.as_normative_docs: 197717
